tools/build/wheel/split_wheel.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import os
+import sys
+import subprocess
+
+import logging
+from pathlib import Path
+from typing import List, Dict, Optional
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+# TODO: Add a function to install dependencies from requirements.txt
+# TODO: Add a function to do submodule setup
+
+
+def get_git_root() -> Path:
+    """
+    Get the current root directory of a Git repository.
+    Returns:
+        str: The path to the Git repository root.
+    """
+    try:
+        # Run the 'git rev-parse --show-toplevel' command
+        git_root = (
+            subprocess.check_output(["git", "rev-parse", "--show-toplevel"])
+            .decode("utf-8")
+            .strip()
+        )
+        return Path(git_root)
+    except subprocess.CalledProcessError:
+        # If the command fails, it means we're not in a Git repository
+        logger.warning("No git repository detected, assuming you are at the root")
+        return Path(".")
+
+
+def get_setup_py() -> Path:
+    return get_git_root() / Path("setup.py")
+
+
+def setup_py(cmd_args: List[str], extra_env: Optional[Dict[str, str]] = None):
+    if extra_env is None:
+        extra_env = {}
+    cmd = [sys.executable, str(get_setup_py()), *cmd_args]
+    logger.debug("+ %s", " ".join(cmd))
+    subprocess.run(
+        cmd,
+        # Give the parent environment to the subprocess
+        env={**os.environ, **extra_env},
+        check=True,
+    )
```

Comment:
```suggestion
    subprocess.check_call(
        cmd,
        # Give the parent environment to the subprocess
        env={**os.environ, **extra_env},
    )
```

Reply:
`subprocess.run(check=true)` is actually the supported version of this now.

From: https://github.com/python/cpython/blob/75ef92da291247fd9d21ec26bd22bf8055b5f330/Lib/subprocess.py#L16-L40